app/views/posts/_post.html.erb

```diff
@@ -0,0 +1,12 @@
+<div class="post-box">
+  <%= link_to(user_post_path(@user, post)) do%>
+    <div class="post">
+      <p><strong>Post #<%= post_counter + 1 %></strong></p>
+      <h3><%= post.title %></h3>
+      <p> <%= post.text %> </p>
+    </div>
+  <% end %>
+  <div class="comments">
+    <%= render post.comments %>
+  </div>
```

Comment:
- [ ] Here it is advised to show **5 most recent comments** by using your method `five_recent_comments` which is in your post model. Please fix it.